CONTRIBUTING.md

```diff
@@ -26,50 +26,98 @@ We utilize GitHub issues to track the feature requests as well. If you are certa
 
 We actively welcome your pull requests:
 
-1. Fork the repo and create your branch from [`main`](https://github.com/openvinotoolkit/anomalib/tree/main).
-1. Set up the development environment by following the instructions below.
-1. If you've added code that should be tested, add tests.
-1. If you've changed APIs, update the documentation.
-1. Ensure the test suite passes.
-1. Make sure you own the code you're submitting or that you obtain it from a source with an appropriate license.
-1. Add a summary of the changes to the [CHANGELOG](https://github.com/openvinotoolkit/anomalib/blob/main/CHANGELOG.md) (not for minor changes, docs and tests).
-1. Issue that pull request!
+###  Getting Started
 
-### Setting up the development environment and using pre-commit
+#### 1. Fork and Clone the Repository
 
-To setup the development environment, you will need to install development requirements, as well as the base requirements of the library.
+First, fork the Anomalib repository by following the GitHub documentation on [forking a repo](https://docs.github.com/en/enterprise-cloud@latest/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo). Then, clone your forked repository to your local machine and create a new branch from `main`.
 
-```bash
-conda create -n anomalib_dev python=3.10
-conda activate anomalib_dev
-pip install -r requirements/base.txt -r requirements/dev.txt
-pre-commit install
-```
+#### 2. Set Up Your Development Environment
+
+Set up your development environment to start contributing. This involves installing the required dependencies and setting up pre-commit hooks for code quality checks.
```

Comment:
For completeness we could add here that this guide will use conda (and assumes the developer has conda installed on their system), but the steps are the same for other env management tools.